### DISCUSSION TITLE
How to hide popup cmd window during installation

### DISCUSSION THREAD

**Discussion by seidhkona** [wixtoolset/issues#7075] [Questions]
I've searched everywhere, but couldn't find any solution on how I can hide popup cmd windows during installation. My CA executes a .batch file, here is the code:
```
<CustomAction Id="InstallService" ExeCommand="[INSTALLDIR]install-service.bat" Directory="INSTALLDIR" Execute="deferred" Impersonate='no' Return='ignore' />
       
<InstallExecuteSequence>
<Custom Action='InstallService' Before="InstallFinalize">NOT (REMOVE~="ALL")</Custom>
</InstallExecuteSequence>
```

I would like to hide the popup window opened by install-service.bat command.

I'm coming from AdvancedInstaller world where it could be solved with one click of a checkbox, so manual configuration is something I'm getting used to.. Any suggestions [with working example] would be greatly appreciated!!!

**Comment by robmen** [comment]
Why did you switch from AdvancedInstaller to WiX?
